pkg/virt-handler/vm.go

```diff
@@ -541,7 +541,6 @@ func (d *VirtualMachineController) processVmShutdown(vm *v1.VirtualMachine, doma
 			return err
 		}
 	}
-	d.recorder.Event(vm, k8sv1.EventTypeNormal, v1.Deleted.String(), "VM stopping")
```

Comment:
"stopping" directly maps to libvirt "destroy" being called.  I think it's helpful to retroactively see when the destroy was called and when the vm actually stopped. 

The "stopping" event could get moved up into that if statement though right under a successfull 'KillVirtualMachine' invocation.

Reply:
We potentially would then report it multiple times too and we don't have an idea when it was delivered. We can change the virt-launcher reply struct, so that it can attach identifyable events. Then we can log them properly. I agree this event is interesting.